modules/custom/openy_search/openy_google_search/openy_google_search.install

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * @file
+ * Module installation file.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function openy_google_search_install() {
+
+  if (\Drupal::service('path.alias_storage')->aliasExists('/search', 'en')) {
+    return;
+  }
+ $installed = \Drupal::service('module_installer')->install(['openy_prgf_google_search']);
+  if (!$installed) {
+    return;
+  }
+
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $node_storage = \Drupal::entityTypeManager()->getStorage('node');
+
+  $search_paragraph = $paragraph_storage->create([
+    'type' => 'google_search',
+    'field_prgf_block' => [
+      0 => [
```

Comment:
I think we can remove `0` key here.
Because `field_prgf_block' => [[]]`  is the same as `'field_prgf_block' => [0 => []]`